src/v/cluster/config_manager.cc

```diff
@@ -446,6 +447,15 @@ apply_local(cluster_config_delta_cmd_data const& data, bool silent) {
         try {
             auto val = YAML::Load(val_yaml);
             auto& property = cfg.get(u.first);
+
+            for (const auto& u : data.upsert) {
```

Comment:
This would appear to log the updates multiple times, I don't think the loop is desired.

It also looks like `val_yaml` is logged in the catch handlers.

Reply:
Good catch(es), that loop was a copy paste from where we used to log.

Updated error handlers to also use the redaction helper.